src/lint/koEsLintLinter.py

```diff
@@ -0,0 +1,127 @@
+from xpcom import components
+from koLintResult import KoLintResult, SEV_ERROR, SEV_WARNING, SEV_INFO
+from koLintResults import koLintResults
+import os
+import logging
+import tempfile
+import process
+import koprocessutils
+import which
+import json
+
+Cc = components.classes
+Ci = components.interfaces
+
+log = logging.getLogger("koEsLintLinter")
+log.setLevel(logging.DEBUG)
+
+
+class KoEsLintLinter(object):
+    _com_interfaces_ = [Ci.koILinter]
+    _reg_clsid_ = "{00CEDD2B-BE9E-4FD6-A9DE-5405B5B4503D}"
+    _reg_contractid_ = "@addons.defman.me/koEsLintLinter;1"
+    _reg_categories_ = [
+        ("category-komodo-linter", 'JavaScript&type=eslint'),
+        ("category-komodo-linter", 'Node.js&type=eslint')
```

Comment:
I read on the website that it also supports JSX, that should probably be covered here too?